### DISCUSSION TITLE
Table is not centered when text is

### DISCUSSION THREAD

**Discussion by PierreMarchand20** [pradyunsg/furo#768] [Potential Issue]
When using `:align: center` in `.. list-table::`, the table is flushed to the left and the text is centered. Without this option, the text is flushed to the left but the table is centered (see below).
![image](https://github.com/pradyunsg/furo/assets/15056323/71d60916-2612-4609-b399-95ca58ad092a)
![image](https://github.com/pradyunsg/furo/assets/15056323/84d4e67d-0683-4785-b4d3-0c6f6893b19c)

In the first case, the html element is 
![image](https://github.com/pradyunsg/furo/assets/15056323/4318ae9a-0c0d-40f3-8fd7-368eb1299b05)

while in the second case, 
![image](https://github.com/pradyunsg/furo/assets/15056323/138b90c9-c3b0-485b-846b-5c2ce5e59f89)

Is it possible to have a centered table with centered text ?

**Comment by pradyunsg** [comment]
Huh. That's definitely a bug.
